README.md

```diff
@@ -134,11 +134,11 @@ const queryLatest = {
 }
 
 function Component() {
-  const { appConfig } = useSiteMetadata()
+  const { siteMetadata } = useMarketMetadata()
   const [result, setResult] = useState<QueryResult>()
 
   useEffect(() => {
-    if (!appConfig.metadataCacheUri) return
+    if (!siteMetadata?.appConfig.metadataCacheUri) return
```

Comment:
so nested :-) Awesome, thanks for doing this context merge in here. As we are accessing `appConfig` a lot think would make sense moving it up one level into `MarketMetadataProviderValue` data structure so `SiteMetadata` is then left to be only the content from json files. But that's like, super bonus. Would have advantage that `appConfig` would be always present from the beginning as we can directly import the file and add it to the context provider value so never a need for optional chaining then

Reply:
also remembered dealing with importing data and merging in extra method is a remnant from Gatsby where we had to get this data with graphql and then pass it through. With Next.js we could ditch all this when we deal with import of json & js files which are present at build time, and just directly import in `@context/MarketMetadata/index.tsx` and add to context value

Reply:
ok, this should be the final version